src/libpgmoneta/message.c

```diff
@@ -2380,6 +2380,10 @@ pgmoneta_receive_archive_stream(SSL* ssl, int socket, struct stream_buffer* buff
       pgmoneta_consume_copy_stream_end(buffer, msg);
    }
 
+   // Initialize the token bucket to limit the rate to 512 MB/s
+   // Capacity is set to 1 GB
+   struct token_bucket* bucket = pgmoneta_token_bucket_init(1024 * 1024 * 1024, 512 * 1024 * 1024, 1);
```

Comment:
No, default is full speed.

This need to come from a configuration parameter or multiple